app/views/pipeline_outputs/show.html.erb

```diff
@@ -50,6 +52,14 @@
           <tr>
             <td><%= element.tax_id %></td>
             <td><%= element.name %></td>
+            <td><% case element.tax_level
```

Comment:
This might be a bit cleaner like:

```
<td><%= case element.tax_level
                 when 1; 'species'
                 when 2; 'genus'
                 end %>
</td>
```

or something like that
and similar below

Reply:
cool